admin_guide/quota.adoc

```diff
@@ -142,30 +157,36 @@ usage is incremented immediately upon the request to create or modify the
 resource.
 
 When you delete a resource, your quota use is decremented during the next full
-recalculation of quota statistics for the project. When you delete resources, a
-link:#configuring-quota-sync-period[configurable amount of time] determines how
-long it takes to reduce quota usage statistics to their current observed system
-value.
+recalculation of quota statistics for the project.
+// end::admin_quota_enforcement[]
+A link:#configuring-quota-sync-period[configurable amount of time] determines
+how long it takes to reduce quota usage statistics to their current observed
+system value.
 
 If project modifications exceed a quota usage limit, the server denies the
 action, and an appropriate error message is returned to the user explaining the
 quota constraint violated, and what their currently observed usage stats are in
 the system.
 
 [[requests-vs-limits]]
-=== Requests vs Limits
+== Requests vs Limits
 
-When allocating compute resources, each container may specify a request and a limit value
-for either CPU or memory. The quota can be configured to quota either value.
+// tag::admin_quota_requests_vs_limits[]
+When allocating
+link:../dev_guide/compute_resources.html#dev-compute-resources[compute
+resources], each container may specify a request and a limit value for either
```

Comment:
Suggest: "each container may specify both *request* and *limit* values", for parallelism and to highlight terminology on first use.